cmd/init.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright © 2024 Josh Holbrook <josh.holbrook@gmail.com>
+*/
+package cmd
+
+import (
+	"github.com/jfhbrook/dosapp/config"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+)
+
+var initCmd = &cobra.Command{
+	Use:   "init",
+	Short: "Initialize dosapp's configuration",
+	Long:  `Initialize dosapp's main configuration.`,
+	Run: func(cmd *cobra.Command, args []string) {
```

Comment:
I discovered a lot of bugs in bash dosapp while writing this and the root command. Is there a good way I can test this? Tests are challenging because the work is 80% accomplished through filesystem operations and command execution.